specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1128 @@
+            "type": "string"
+          }
+        },
+        "heartbeatTimeoutInSecond": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "SparkJobLog": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "from": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "size": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "log": {
+          "x-ms-client-name": "logLines",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SparkJobState": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "state": {
+          "type": "string"
+        }
+      }
+    },
+    "SparkStatementCollection": {
+      "type": "object",
+      "properties": {
+        "statements": {
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SparkStatement"
+          }
+        }
+      }
+    },
+    "SparkStatement": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "code": {
+          "type": "string"
+        },
+        "state": {
+          "type": "string"
+        },
+        "output": {
+          "$ref": "#/definitions/SparkStatementOutput"
+        }
+      }
+    },
+    "SparkStatementOutput": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "type": "string"
+        },
+        "execution_count": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "data": {
+          "type": "object"
+        }
+      }
+    },
+    "SparkStatementRequest": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "kind": {
+          "type": "string"
+        }
+      }
+    },
+    "SparkStatementCancellationResult": {
+      "type": "object",
+      "properties": {
+        "msg": {
+          "x-ms-client-name": "cancelMessage",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "requestedByParameter": {
+      "name": "X-Requested-By",
+      "x-ms-parameter-location": "client",
+      "x-ms-client-name": "requestedBy",
+      "description": "Add default value for X-Requested-By in header.",
+      "in": "header",
+      "required": false,
+      "default": "ambari",
+      "type": "string"
+    },
+    "endpointParameter": {
+      "name": "endpoint",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-skip-url-encoding": true,
+      "description": "The cluster dns name against which the job management is to be.",
```

Comment:
>The cluster dns name [](start = 22, length = 20)

Maybe this description needs to be updated too.

Reply:
Fixed it.